main.go

```diff
@@ -47,14 +46,22 @@ func index(w http.ResponseWriter, r *http.Request) {
 		http.Error(w, err.Error(), http.StatusInternalServerError)
 		return
 	}
-	w.Write(file)
+	_, err = w.Write(file)
+	if err != nil {
+		fmt.Println(err)
```

Comment:
I think we should be using `handleHTTPError` in these error handlers, WDYT?

Reply:
I'm not sure what one can do to actually induce an error in `w.Write`, so I don't know what will happen in the real world, but `w.Write` will set `StatusOK` before it actually writes the bytes*, and then setting the status a second time results in `http: superfluous response.WriteHeader call from main.index (main.go:41)`, so I generally try to avoid doing anything with the ResponseWriter once I've called Write. I haven't been able to find any guidance on this online.

\* except in the case of connection hijacking

Reply:
That's a good point!